fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -191,6 +191,7 @@ public class SessionVariable implements Serializable, Writable, Cloneable {
     public static final String MAX_PARALLEL_SCAN_INSTANCE_NUM = "max_parallel_scan_instance_num";
     public static final String ENABLE_INSERT_STRICT = "enable_insert_strict";
     public static final String INSERT_MAX_FILTER_RATIO = "insert_max_filter_ratio";
+    public static final String INSERT_TIMEOUT = "insert_timeout";
```

Comment:
If the user has set query_timeout, how to maintain compatibility when the user upgrades?

Reply:
add `insert_timeout` session variable is to make it easier for users to change.